src/components/useraccount/MyOrders.vue

```diff
@@ -96,6 +98,14 @@ export default {
     me: null,
   }),
 
+  methods: {
+    translateStatus(state) {
+      if (state) {
+        return this.$t(state);
+      } return '-';
```

Comment:
If you prefer one-liner style, here is a good place to use it 😉 
`return state ? this.$t(state) : '-';`

Reply:
haha yeah I like it more